### DISCUSSION TITLE
Issue tracking (ticket mishandling)

### DISCUSSION THREAD

**Discussion by Chealer** [ReactiveX/rxjs#6576] [General]
Greetings,
I am new to RxJS, and as such, I have been trying to familiarize myself using the documentation. Which, as many of you probably know, could use quite some work.

There are several issues in the documentation I would volunteer to report... except that I already reported a few, and the result is so far appalling. I suppose the following speaks for itself:
#6568 
#6569
#6570
#6571 

This would in most circumstances be more than enough to piss me off, except the above outcomes can all be attributed to a single individual I need not naming, so I am not giving up just yet.

Is the above representative of the RxJS community's standards, or are issues usually tracked reasonably well?

**Comment by voliva** [comment]
The way I see it, this is an open-source community. The core team is organizing a library that ~~thousands~~ millions of projects use and they are doing this for free.

I think we, as consumers of this library, should be as helpful and understanding to them as possible.

Answering your specific question, I think they do a great job. While I agree that you won't get an answer as if they were in 24/7 support (because again, it's not a service we're talking about here), if you report issues that effect the behaviour (i.e. they are more urgent) they are swift enough to answer. At least this is my experience and the feeling I got when working with this community.

If they want to restrict the Github [Issues] tab to only core bugs and core team discussions, so be it. I personally don't mind, it's just another way of organising a repo.

> There are several issues in the documentation I would volunteer to report... except that I already reported a few, and the result is so far appalling.

I really apreciate these contributions you've made, and even if in one of them I've expressed that I think it doesn't need changing, that's just my opinion. I'm just one of the thousands of devs that use this library, and English is not my first (nor second) language, so maybe I should've been more careful with my wording, but I like that it brings up a discussion and others can participate in.

For the one you're referencing but not naming, I'd say don't take it as if you're getting ignored. Your issue has just gone through triage and it's now organized. I've also had some disagreement some time with an issue when it got through triage (in my case it was marked as duplicate when in my opinion it wasn't) I just politely expressed my view on it and moved on. Again, they're doing this for free.

Also keep in mind that although those issues are closed, they've been moved to their respective discussions. When you go to "issues" at the top there's one pinned **"Before filing an issue: github discussions now available"** where it explains that only core bugs should go to issues, the rest goes to discussions (and they have many categories in here). If you're seeing one of your issues moved into discussions, then I would post the next issues of that same nature to the appropiate place.

Again, don't take this answer as offensive please, I'm just one random rxjs consumer and I'm giving my opinion in a discussion of an open-source community. Everyone who contributes to rxjs (which includes you ofc) has my respect.

**Comment by Chealer** [comment]
> [...]
> Is the above representative of the RxJS community's standards, or are issues usually tracked reasonably well?

I assumed the former from the limited reactions here, so I for one [announced my departure](https://www.philippecloutier.com/blogpost178) from this project.
